source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsChooseWalletDialog.scss

```diff
@@ -9,6 +9,21 @@
         background: transparent;
       }
     }
+
+    .WalletsDropdownOption_topRow .WalletsDropdownOption_syncing > div .LoadingSpinner_icon svg path {
```

Comment:
This probably belongs to `WalletsDropdownOption.scss`

Reply:
Its an override only for DelegationStepsChooseWalletDialog as this one has different loading spinner color then on other screens.